traveler-swift-core/TravelerKit/Models/Order.swift

```diff
@@ -34,18 +36,18 @@ public struct Order: Decodable {
     enum CodingKeys: String, CodingKey {
         case id = "id"
         case total = "amount"
-        case orderNumber = "orderNumber"
+        case referenceNumber = "referenceNumber"
         case products = "products"
         case status = "status"
-        case createdDate = "createdAt"
+        case createdDate = "createdOn"
     }
 
     public init(from decoder: Decoder) throws {
         let container = try decoder.container(keyedBy: CodingKeys.self)
 
         self.id = try container.decode(String.self, forKey: .id)
         self.total = try container.decode(Price.self, forKey: .total)
-        self.orderNumber = try container.decode(String.self, forKey: .orderNumber)
+        self.referenceNumber = try container.decode(String?.self, forKey: .referenceNumber) ?? nil // TODO: Double check whether referenceNumber can actually be false when Alex returns
```

Comment:
reference numbers can be empty / null this is a string on the backend

Reply:
okay cool 👍